frontend/scripts/react-components/shared/grid-list-item/grid-list-item.component.jsx

```diff
@@ -16,21 +16,25 @@ function GridListItem(props) {
     enableItem,
     disableItem,
     onInfoClick,
-    isInfoActive
+    onHover,
+    isInfoActive,
+    variant
   } = props;
   if (!item) return <b style={style} />;
   const onClick = isActive && disableItem ? disableItem : enableItem;
   const testId = item.name && item.name.split(' ').join('-');
+  const onHoverProp = onHover ? () => onHover(item) : {};
```

Comment:
I don't love this pattern because when I read `{...onHoverProp}` I don't know if I'm reading a prop or a local variable or what does it contain. I prefer a ternary `onHover={onHover ? () => onHover(item : undefined}`